concepts/05 UI Components/NumberBox/00 Getting Started with NumberBox/00 Getting Started with NumberBox.md

```diff
@@ -0,0 +1,9 @@
+#include tutorials-intro-installationnote
+
+The NumberBox component allows users to enter a number or specify it with the spin buttons.
+
+This tutorial shows how to add NumberBox to a page and configure the component's core settings. As a result, you will create the following UI component:
```

Comment:
```suggestion
This tutorial shows how to add a NumberBox to a page and configure the component's core settings. The image below shows the resulting UI component:
```

Reply:
There will be an online sample instead of image